openvdb/openvdb/tools/LevelSetFilter.h

```diff
@@ -160,7 +160,7 @@ class LevelSetFilter : public LevelSetTracker<GridT, InterruptT>
 
         Filter(LevelSetFilter* parent, const MaskType* mask) : mParent(parent), mMask(mask) {}
         Filter(const Filter&) = default;
-        virtual ~Filter() {}
+        ~Filter() {}
```

Comment:
I should say this is still being tested by the CI. The error that was firing: "Undefined symbols for architecture x86_64: "vtable for openvdb::...::LevelSetFilter, openvdb::...::Filter", referenced from vdb_tool::Tool::offsetLevelSet()"". I don't know why the destructor was marked virtual in the first place since I do not see any other class inheriting from it. I will make a comment again if the CI passes with this fix.

Reply:
great catch!!! This has a bug for a long time